### DISCUSSION TITLE
Deactivate the extra login step when entering invoices etc.

### DISCUSSION THREAD

**Discussion by VOOM108** [kimai/kimai#4530] [Questions & Answers]
Is it possible to do that? In a one-person-homeoffice setting, it does not make much sense, to have to provide the password again and again after a timeout to enter specific areas of kimai. Is there an option or hack to turn that off?

**Comment by kevinpapst** [comment]
You see it the wrong way around.
The remember me mode allows you to use Kimai for a week, even if your session expired. It will restart it without the need to re-login:
https://www.kimai.org/documentation/users.html#login

Your system configuration is the real troublemaker. 
Configure your PHP to use longer session lifetimes and you will not have to re-login all the time.
The default session lifetime of PHP is 14 minutes, which is basically upon every click in Kimai.
Yes, I understand your point, but technically the session is gone and this is not managed by Kimai.

Sensitive areas will not be opened-up to the `remember me` mode.

**Comment by VOOM108** [comment]
I see

> Sensitive areas will not be opened-up to the remember me mode.

A switch to allow that would basically have been what I was hoping to find...  ;)

**Reply by kevinpapst** [reply]
This feature is deeply buried in the software and cannot be dynamically changed.

From my POV Kimai does you a favor by allowing access, even though your session was already expired.
Your hosting environment is misconfigured if you have to re-login often!

**Comment by VOOM108** [comment]
It is a very complex unique password only saved in Bitwarden. It is not so much about having to relogin several times because of a timeout, but about unlocking the vault, unlocking it in the default browser, and unlocking it again in the installed kimai "webapp" based on chrome. But of course it is just an inconvenience, and I hear you that this is the way it works and will not change :)

**Reply by kevinpapst** [reply]
I hear you and it is really time that we can get rid of password managers somehow ... but what I am asking is: why you are trying to change the software, instead of the session_timeout setting for PHP, which is done in two minutes. Or don't you have access to it?

**Comment by VOOM108** [comment]
Unless I could set session timeouts to a week or so, it would not really help. And I am not sure about the implications for other stuff installed on the cloud server.

**Reply by kevinpapst** [reply]
Yes, you can set it to any number in seconds. 
And you can configure it per host, so only for Kimai. 
You can overwrite PHP ini settings for each virtual host.
